examples/js/controls/OrbitControls.js

```diff
@@ -317,9 +317,7 @@ THREE.OrbitControls = function ( object, domElement ) {
 		return function panUp( distance, objectMatrix ) {
 
 			v.setFromMatrixColumn( objectMatrix, 0 ); // get X column of objectMatrix
-
-			v.applyAxisAngle( new THREE.Vector3( 0, 0, 1 ), Math.PI / 2 );
-
+			v.applyAxisAngle( new THREE.Vector3( 0, 1, 0 ), Math.PI / 2 );
```

Comment:
This logic is not correct if `camera.up` is changed.

Also, we need to avoid instantiating a new `Vector3` here. This method can be called many times per second.

Also, this PR will change the behavior for current users. It would be better to have the panning method as an option.